Source/Lib/Codec/EbUtility.c

```diff
@@ -3,26 +3,14 @@
 * SPDX - License - Identifier: BSD - 2 - Clause - Patent
 */
 
-
-#include "EbUtility.h"
-#ifdef _WIN32
-#include "EbDefinitions.h"
-//#if  (WIN_ENCODER_TIMING || WIN_DECODER_TIMING)
 #include <time.h>
-//#endif
-#elif __linux__
-#include <stdio.h>
-#include <stdlib.h>
-#include "EbDefinitions.h"
+#ifndef _WIN32
 //#if   (LINUX_ENCODER_TIMING || LINUX_DECODER_TIMING)
```

Comment:
also remove //#if   (LINUX_ENCODER_TIMING || LINUX_DECODER_TIMING) and //#endif?

Reply:
Sure, wasn't sure if this should've been removed or not